exalearn-api/ExaLearn.Dal/Repositories/AssignTestRepository.cs

```diff
@@ -28,5 +31,11 @@ public async Task<IList<AssignTest>> GetHrExpiredAssignedTestByIdAsync(int hrId)
         {
             return await _appDbContext.AssignTests.Include(x => x.User).Where(x => (x.AssignerId == hrId) && x.IsExpired).ToListAsync();
         }
+
+        public void ArchiveExpiredAssignTest()
+        {
+            _appDbContext.Database.ExecuteSqlRaw("call archiveexpiredassigntest({0})", DateTime.UtcNow);
+            _appDbContext.Database.OpenConnection();
```

Comment:
Execute and then OpenConnection? Is this method necessary

Reply:
can you executesqlraw async?

Reply:
ExecuteSqlRawAsync maybe, if not okey

Reply:
Hangfire is unevenly async(another thread), and it cannot run async methods

Reply:
> Execute and then OpenConnection? Is this method necessary

deleted

Reply:
![image](https://user-images.githubusercontent.com/71830071/129389994-73b60a31-e909-40e5-95ba-ffad2d7a7915.png)
![image](https://user-images.githubusercontent.com/71830071/129390026-523b0afb-4aad-4b7f-aacb-07130a47ba6f.png)